crates/kernels/vulkan-and-cpu/src/kernel.rs

```diff
@@ -172,6 +165,14 @@ impl<'kernel> Kernel<'kernel> {
             self.constants.reserved_rings_per_band,
         );
 
+        // This can't be placed on `Self` because on Vulkan any writes to it then cause an error
```

Comment:
This is an interesting bug.... I wonder if it has something to do with what descriptor the struct is loaded in on.... I would agree that it is reportable.